<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QMainWindow" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="inputFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoWidget" name="inputFrame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoWidget" name="outputFrame">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VideoWidget" name="outputFrame_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="equalizerSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="saturationSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hueSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="toleranceSilder">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="hueTextLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue</string>
    </property>
   </widget>
   <widget class="QLabel" name="toleranceTextLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerance</string>
    </property>
   </widget>
   <widget class="QLabel" name="saturationTextLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="equalizerTextLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Equalizer</string>
    </property>
   </widget>
   <widget class="QLabel" name="hueValue">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="toleranceValue">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="saturationValue">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="equalizerValue">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="medianCheck">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="openingCheck">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Opening</string>
    </property>
   </widget>
   <widget class="QLabel" name="xLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="zLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xValue">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="yValue">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="zValue">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="midiOutTextLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MIDI out</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="xSlider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>640</number>
    </property>
    <property name="value">
     <number>320</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ySlider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>480</number>
    </property>
    <property name="value">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zSlider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>640</number>
    </property>
    <property name="value">
     <number>320</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionAdd_Camera"/>
    <addaction name="actionAdd_Camera_2"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionPlay"/>
   </widget>
   <addaction name="menuSelect"/>
   <addaction name="menuPlay"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Camera">
   <property name="text">
    <string>Add Camera 1</string>
   </property>
  </action>
  <action name="actionAdd_Camera_2">
   <property name="text">
    <string>Add Camera 2</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QLabel</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>hueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoPlayer</receiver>
   <slot>updateParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>903</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toleranceSilder</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoPlayer</receiver>
   <slot>updateParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoPlayer</receiver>
   <slot>updateParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>938</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>956</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>equalizerSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoPlayer</receiver>
   <slot>updateParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>943</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>medianCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>VideoPlayer</receiver>
   <slot>updateParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openingCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>VideoPlayer</receiver>
   <slot>updateParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>864</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateParameters()</slot>
 </slots>
</ui>
